tuftool/src/refresh.rs

```diff
@@ -75,124 +68,43 @@ pub(crate) struct RefreshArgs {
 
 impl RefreshArgs {
     pub(crate) fn run(&self) -> Result<()> {
-        if let Some(jobs) = self.jobs {
```

Comment:
It won't let me comment on the new location of this, but it made me think - the comment on `jobs` says it defaults to the number of cores, but there's no structopt-level default, and here we're checking if the argument was given... are we actually getting parallelism?  Do we need a default in the args struct?

Reply:
Good question. Per the [`rayon` FAQ](https://github.com/rayon-rs/rayon/blob/master/FAQ.md), it will spawn the same number of threads as cores. It does this by default when you use any of the parallel iterator methods it provides, which we do below in `build_targets()`.

If a user specifies a specific number of jobs, we use `rayon::ThreadPoolBuilder` to override the default.

Reply:
Ah, sorry, I was silly and forgot that rayon creates a global thread pool by default.  This is just configuring it, if the user wanted a different number of threads.

Reply:
I'll add a comment for future readers.